### PR TITLE
Mapping idxs of structureIC

### PR DESCRIPTION
**Pull request by MarioOcampo**
This PR updates the index of the streamlines in the groups (`structureIC`) if streamlines were not considered in the creation of the dictionary.

### CODE REVIEW

File: commit/solvers.py [hunk 1]
```diff
@@ -121,6 +121,30 @@ def init_regularisation(commit_evaluation,
     regularisation['lambdaISO'] = float( lambdas[2] )
 
     # Solver-specific fields
+    # Check if idxs in groups need to be updated
+    if(0 in commit_evaluation.DICTIONARY['TRK']['kept']):
+        dictionary_TRK_kept = commit_evaluation.DICTIONARY['TRK']['kept']
+
+        idx_in_kept = np.zeros(dictionary_TRK_kept.size, dtype=np.int64) - 1
```
Comment (daducci) [inline commit/solvers.py:128]: Do we need `np.int64` or we could be happy with `np.int32`?

Comment (daducci) [reply commit/solvers.py:128]: Write "..np.int32) - 1 # -1 is used to flag indices for removal"

File: commit/solvers.py [hunk 2]
```diff
@@ -121,6 +121,30 @@ def init_regularisation(commit_evaluation,
     regularisation['lambdaISO'] = float( lambdas[2] )
 
     # Solver-specific fields
+    # Check if idxs in groups need to be updated
+    if(0 in commit_evaluation.DICTIONARY['TRK']['kept']):
```
Comment (daducci) [inline commit/solvers.py:125]: Change to `if ( 0 in commit_evaluation.DICTIONARY['TRK']['kept'] ) :`

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,13 +2,14 @@
...
 
-## [1.5.0] - 2021-05-07
+## [1.5.0] - 2021-06-19
 
 ### Added
 - The possibility to specify a voxel confidence map
 
 ### Fixed
 - Set license: BSD-3-Clause License 
+- Idx in groups if there are streamlines no kept in the dictionary
```
Comment (daducci) [inline CHANGELOG.md:12]: Change to `Indices of groups in case of streamlines that were pre-filtered during trk2dictionary`

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,13 +2,14 @@
 # Change Log
 All notable changes to COMMIT will be documented in this file.
 
-## [1.5.0] - 2021-05-07
+## [1.5.0] - 2021-06-19
 
 ### Added
 - The possibility to specify a voxel confidence map
```
Comment (daducci) [inline CHANGELOG.md:8]: Change to `Possibility to specify a voxel confidence map`

File: commit/solvers.py (hunk 2 above)
Comment (MarioOcampo) [reply commit/solvers.py:125]: I'll remove the "(  )" and the space before ":"

**Pull request merged**